Helper/Cacher.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace Reach\Payment\Helper;
+
+//use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\App\Cache\Type\FrontendPool;
+use Magento\Framework\Cache\Frontend\Decorator\TagScope;
+
+use Magento\Framework\Serialize\SerializerInterface;
+
+
+/**
+ * Helper Class
+ */
+//class Cacher extends AbstractHelper
+class Cacher extends TagScope
+{
+    /**
+     *  @var \Psr\Log\LoggerInterface
+     */
+    protected $_logger;
+
+    /**
+     * @var SerializerInterface
+     */
+    private $serializer;
+
+    const REACH_CACHE_ID = 'reach_payment_custom_cache';
+    const REACH_CACHE_TAG = 'REACH_PAYMENT_CUSTOM_CACHE';
```

Comment:
Is needed for clearing or flushing a specific cache instead of clearing everything (or something with wider scope).